alfresco-dynamic-extensions-repo/build.gradle

```diff
@@ -27,7 +27,7 @@ subprojects {
         ampLib(project(":annotations-runtime:scheduler-quartz-1")) { transitive = false }
         ampLib(project(":annotations-runtime:scheduler-quartz-2")) { transitive = false }
 
-        ampLib(project(":webscripts")) { transitive = false }
```

Comment:
How is this related? What does is change / fix?

Reply:
Added comments, see on the right